flag_engine/identities/schemas.py

```diff
@@ -1,21 +1,26 @@
 import typing
 
-from marshmallow import EXCLUDE, fields, post_dump
+from marshmallow import EXCLUDE, Schema, fields, post_dump
 
 from flag_engine.features.schemas import FeatureStateSchema
 from flag_engine.identities.models import IdentityModel, TraitModel
 from flag_engine.utils.marshmallow.fields import ListOrDjangoRelatedManagerField
 from flag_engine.utils.marshmallow.schemas import LoadToModelSchema
 
 
-class TraitSchema(LoadToModelSchema):
+class BaseTraitSchema(Schema):
     trait_key = fields.Str()
     trait_value = fields.Str()
 
     class Meta:
         model_class = TraitModel
 
 
+class TraitSchema(LoadToModelSchema, BaseTraitSchema):
```

Comment:
```suggestion
class TraitSchema(LoadToModelMixin, BaseTraitSchema):
```

Reply:
Note that you might need to order the base classes in a specific order. Make sure you test the load works as we expect (possibly by waiting for the tests from the other PR).